src/components/dialogs/connectivity-edition.js

```diff
@@ -232,7 +233,11 @@ const ConnectivityEdition = ({
                         id="bus"
                         disabled={!voltageLevel}
                         options={busOrBusbarSectionOptions}
-                        getOptionLabel={(bbs) => bbs?.id || ''}
+                        getOptionLabel={(bbs) => {
```

Comment:
Isn't it the same as before, but with more code ?

Reply:
Not exactly, we can't set `busOrBusbarSection` to null to clear the field anymore. Then I replaced null with ' ' (empty string) and manage this case in getOptionLabel (which is called in this case) to clear the field.